src/renderer.rs

```diff
@@ -643,3 +643,121 @@ pub fn ray_intersect(
         None
     }
 }
+
+///
+/// Representation of a specific geometry instance
+///
+pub struct GeometryInstance {
```

Comment:
Yeah, it used to fit the `Geometry Instance` name better, but has since been adjusted away from that. However, `ray_intersect_instance` also returns that type, so maybe `IntersectionResult` would be good?